### PR TITLE
Camera ready simulator

### PR DESCRIPTION
**Pull request by GindaChen**

### CODE REVIEW

File: distserve/evaluation/docs/README-AE.md [hunk 1]
```diff
@@ -179,20 +179,17 @@ bash /app/distserve/distserve/evaluation/ae-scripts/e2e/plot-fig-10.sh
...
-- Choose `GPU Count`: 1 GPU is sufficient. In fact, the ablation experiment can be run on CPU instances, but the template we provide requires GPU environment.
-- Click `Deploy On-Demand`: If the button is grey, it means this resource is not currently available.
 
-TODO: @cjd plot to pdf instead of png
 
-Execute xxx
+*Compute Time: 5 min*
+
+The abalation study is CPU-only. We preferred you allocate `RTX3090` or `L40S` where 32 vCPU instance is available.
+
```
Comment (PKUFlyingPig) [inline distserve/evaluation/docs/README-AE.md:187]: "When you successfully log into the machine, execute"

File: distserve/evaluation/docs/README-AE.md [hunk 2]
```diff
@@ -179,20 +179,17 @@ bash /app/distserve/distserve/evaluation/ae-scripts/e2e/plot-fig-10.sh
...
-- Click `Change Template` and choose `DistLLM-AE-GPU`.
-- Choose `GPU Count`: 1 GPU is sufficient. In fact, the ablation experiment can be run on CPU instances, but the template we provide requires GPU environment.
-- Click `Deploy On-Demand`: If the button is grey, it means this resource is not currently available.
 
-TODO: @cjd plot to pdf instead of png
 
-Execute xxx
+*Compute Time: 5 min*
+
+The abalation study is CPU-only. We preferred you allocate `RTX3090` or `L40S` where 32 vCPU instance is available.
```
[the remark this answers is not in the document]
Comment (GindaChen) [reply distserve/evaluation/docs/README-AE.md:186]: Fixed

File: simdistserve/base/constants.py
[the remark this answers is not in the document]
Comment (GindaChen) [reply simdistserve/base/constants.py:1]: Removed

File: distserve/evaluation/docs/README-AE.md (hunk 1 above)
Comment (GindaChen) [reply distserve/evaluation/docs/README-AE.md:187]: Changed the text

File: simdistserve/README.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+Ideally you should get the following result:
+
+```text
+Best per GPU rate: 1.56
+Best config: pp_cross=1, tp_prefill=2, pp_prefill=1, tp_decode=1, pp_decode=1
+```
+
+## Architecture
+
+The simulator is written on top of `simpy`, a discrete event simulator build natively in Python.
```
Comment (PKUFlyingPig) [inline simdistserve/README.md:53]: built

File: simdistserve/README.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+In the high level, our simulator is composed of the following core components (under the `base` module):
+
+- `Worker`. A worker simulates the behavior of one or more GPU that executes in lock step (i.e. a group of `TP` number of GPUs with tensor parallelism). Worker can be a prefill worker or a decode worker, depending on its assignment in the cluster. Multiple workers can be chained together to form a pipeline. Setting the corresponding `PP` value in the simulator changes the pipeline parallelism for the cluster.
+- `Scheduler`. The scheduler forwards the requests to the workers, with specified arrival rate and process (e.g. Poisson, Gamma, fixed rate).
+- `Cluster`: The cluster is just a wrapper around the workers and scheduler. `DisaggCluster` and `VLLMCluster` are the available options to choose from.
+
+On top of the core components, we implement the embarrassingly parallel search of the optimal parallelism configuration. The search is done by a binary search algorithm that tries to find the optimal per GPU rate that maximizes the goodput of the workload. The algorithm is currently under `benchmarks` module.
```
Comment (PKUFlyingPig) [inline simdistserve/README.md:61]: On top of the core components, we implement the placement algorithm mentioned in our paper that searches for the optimal parallelism configuration. It utilizes a binary search algorithm that tries to find the optimal placement strategy that maximizes the per-GPU goodput.

File: simdistserve/README.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    --model-type "facebook/opt-13b" \
+    --workload sharegpt --backend distserve \
+    --prefill-target 200 --decode-target 100 \
+    --prefill-percentage 90 --decode-percentage 90 \
+    --max-per-gpu-rate 5 \
+    --esp 0.25 \
+    --N 300
+```
+
+This experiment runs ShareGPT with 1 node with 4 A100 (80G) GPU. The model is facebook/opt-13b. We set the prefill SLA
```
Comment (PKUFlyingPig) [inline simdistserve/README.md:39]: `facebook/opt-13b`
change SLA to SLO

File: simdistserve/README.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+to 200 ms, and decode SLA to 100 ms. The `prefill-percentage` and `decode-percentage` indicate the percentage of
+requests to attain, in this case, P90 (90%). The `max-per-gpu-rate` is set to 5 to restrict the binary search runtime,
```
[the remark this answers is not in the document]
Comment (GindaChen) [reply simdistserve/README.md:41]: 5 requests per second per GPU (?)

**Pull request merged**